pkg/services/auth.go

```diff
@@ -90,3 +90,26 @@ func NewAuthorizeRequireGRTJobCode() func(context.Context) (bool, error) {
 		return true, nil
 	}
 }
+
+// NewAuthorizeUserIsIntakeRequesterOrHasGRTJobCode returns a function
+// that authorizes a user as being a member of the
+// GRT (Governance Review Team)
+func NewAuthorizeUserIsIntakeRequesterOrHasGRTJobCode() func(context.Context, *models.SystemIntake) (bool, error) {
+	return func(ctx context.Context, existingIntake *models.SystemIntake) (bool, error) {
```

Comment:
This definitely looks like it works as written. 

There are also some simplifications possible in this code, as well as some nuances around the lifecycle of the closures, but there are definitely nice-to-haves that we can talk about after this PR.